README.md

```diff
@@ -2738,40 +2738,44 @@ See `srfi-26` for detailed description.
 
 #### -not `(pred)`
 
-Take a unary predicate `pred` and return a unary predicate
-that returns t if `pred` returns nil and nil if `pred` returns
-non-nil.
+Return a function that is the logical negation of predicate `pred`.
+When called, the returned variadic function passes its arguments
+to `pred` and returns t if `pred` returns nil, otherwise nil.
 
 ```el
-(funcall (-not 'even?) 5) ;; => t
-(-filter (-not (-partial '< 4)) '(1 2 3 4 5 6 7 8)) ;; => '(1 2 3 4)
+(funcall (-not #'even?) 5) ;; => t
+(-filter (-not (-partial #'< 4)) '(1 2 3 4 5 6 7 8)) ;; => '(1 2 3 4)
+(funcall (-not #'>) 1 2) ;; => t
```

Comment:
I just tried something like this yesterday and was sad it doesn't work with a binary predicate.  Definitely a welcome change.